src/Control/Monad/Bayes/Traced/Static.hs

```diff
@@ -82,8 +82,26 @@ mhStep (Traced m d) = Traced m d'
   where
     d' = d >>= mhTransFree m
 
+-- $setup
+-- >>> import Control.Monad.Bayes.Class
+-- >>> import Control.Monad.Bayes.Sampler.Strict
+-- >>> import Control.Monad.Bayes.Weighted
+
 -- | Full run of the Trace Metropolis-Hastings algorithm with a specified
 -- number of steps. Newest samples are at the head of the list.
+--
```

Comment:
Could you add a description of Sam Staton's bus example? It will be easier to understand with the context.

Reply:
How's this?

Reply:
perfect!